.aptible.yml

```diff
@@ -1,3 +1,4 @@
 before_release:
     - bundle exec rake db:migrate
-    - bundle exec rake db:insert_states
\ No newline at end of file
+    - bundle exec rake db:insert_states
+    - bundle exec rake db:update_tax_return_statuses
```

Comment:
I'd like to see the migration be done as a SQL data migration, or even a Ruby script that lives as a migration. I know we talked about some data migrations being slow, but I think this is a good time to use data migrations because of the the following reasons:

* The app isn't really in a good state if we upgrade the code without migrating the data.
* We have no prod users, so we can't really cause an outage.

I'm open to other ideas, including the approach you have here; this is not a blocker.

If it's helpful, I can try writing the SQL to do the migration, either as a comment on GitHub, or while pairing with you.

Reply:
Awesome! We moved this into a migration file, but still using AR for now.